--- a/media/gpu/vaapi/BUILD.gn
+++ b/media/gpu/vaapi/BUILD.gn
@@ -10,6 +10,11 @@ import("//ui/ozone/ozone.gni")
 
 assert(use_vaapi)
 
+config("vaapi_permissive") {
+  cflags = [ "-fpermissive" ]
+}
+
+
 action("libva_generate_stubs") {
   extra_header = "va_stub_header.fragment"
 
@@ -100,6 +105,8 @@ source_set("vaapi") {
     "//third_party/libyuv",
   ]
 
+  configs += [ ":vaapi_permissive" ]
+
   if (use_x11) {
     configs += [ "//build/config/linux:x11" ]
     deps += [ "//ui/gfx/x" ]
